scripts/main.js

```diff
@@ -0,0 +1,67 @@
+const myLibrary = [];
+
+function Book(author, title, pages, read) {
+  this.author = author;
+  this.title = title;
+  this.pages = pages;
+  this.read = read;
+}
+
+function addBookToLibrary(newBook) {
+  myLibrary.push(newBook);
+}
+
+function displayLibrary(newBook) {
+  const table = document.querySelector('table');
+  const tr = document.createElement('tr');
+  tr.classList.add('book-row');
+
+  const td1 = document.createElement('td');
+  const td2 = document.createElement('td');
+  const td3 = document.createElement('td');
+  const td4 = document.createElement('td');
+  const td5 = document.createElement('td');
+  const removeBtn = document.createElement('button');
+  const td4Btn = document.createElement('button');
+
+  td1.textContent = newBook.author;
+  td2.textContent = newBook.title;
```

Comment:
- [ ] Apart from the constructor function kindly change these two functions to an es6 arrow function syntax.

Reply:
Done